lib/OpenQA/WebSockets/Controller/Worker.pm

```diff
@@ -151,6 +152,14 @@ sub _message {
         log_trace "Received from worker $worker_id worker_status message: " . dumper($json)
           if LOG_WORKER_STATUS_MESSAGES;
 
+        # detect if the websocket server is running too close to its limit and falling behind with status updates
+        my ($last_seen, $now) = ($worker_status->{last_seen}, time);
```

Comment:
I hope getting the time here has only a neglectable performance impact. This just reminds me of 3c42ce17c2b70957215d511166410fbd15a1b04f where it contributed notably to the runtime (but I guess it was more the formatting than getting the time itself).

Reply:
No measurable impact fortunately.